package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@blackbaud/skyux-lib-restricted-view",
+  "version": "1.0.0-rc.1",
+  "description": "Single-page-application for skyux-lib-restricted-view",
+  "main": "bundles/bundle.umd.js",
+  "module": "index.ts",
+  "engines": {
+    "node": ">=6.11.1"
+  },
+  "scripts": {
+    "skyux": "skyux",
+    "test": "skyux test && skyux e2e && skyux build && npm run build",
+    "build": "skyux build-public-library",
+    "start": "skyux serve"
+  },
+  "keywords": [],
+  "author": "Blackbaud",
+  "license": "ISC",
+  "peerDependencies": {
+    "@angular/animations": "^4.3.6",
```

Comment:
You should only have peers listed that are directly imported into your library, and have shared state (i.e., dependencies that you want the consumer to install, but not be bundled within your library).

From a quick look, it appears that you'll only need to list:

```
"@angular/common": ">=4.3.6",
"@angular/core": ">=4.3.6",
"@blackbaud/auth-client": "^2.13.0",
"@skyux/http": "^3.2.0"
```